app/templates/main/select.html

```diff
@@ -38,7 +38,11 @@
             </label>
             <select class="govuk-select" id="to-quarter" name="to-quarter">
                 {% for item in params['to-quarter'] %}
-                    <option value="{{item}}">{{item}}</option>
+                    {% if loop.last %}
+                        <option value="{{item}}" selected>{{item}}</option>
+                    {% else %}
+                        <option value="{{item}}">{{item}}</option>
+                    {% endif %}
```

Comment:
we can do this more concisely: 
```suggestion
                     <option value="{{item}}" {{ 'selected' if loop.last else '' }}>{{item}}</option>
```